lib/riskified_php_sdk/src/Riskified/OrderWebhook/Transport/CurlTransport.php

```diff
@@ -27,6 +27,8 @@ class CurlTransport extends AbstractTransport {
     public $timeout = 10;
     public $dns_cache = true;
 
+    public $requestData;
```

Comment:
It has been added for test purpose. Problem with checkout events has been reported and I we were not able to reproduce this issue. To get some information about it we had to log request and response data at this level. We are aware of that this is not only magento extension, it will be removed when it will be confirmed if everything is ok.